README.md

```diff
@@ -105,11 +105,13 @@ For Windows users, those dependencies are included in GTK. You can find the late
 
 You can then install the latest release of the package using [pypi](https://pypi.org/project/python-doctr/) as follows:
 
+> :warning: Notice that the basic installation is not standalone, as it does not provide a deep learning framework, which is required for the package to run.
+
```

Comment:
One last adjustment: could you rephrase "Notice" to "Please note" ? :pray: 
And move the warning right after the `pip install python-doctr` command?